iroha/src/crypto.rs

```diff
@@ -1,5 +1,9 @@
 use parity_scale_codec::{Decode, Encode};
 use std::fmt::{self, Debug, Formatter};
+use ursa::{
+    keys::PublicKey as UrsaPublicKey,
+    signatures::{ed25519::Ed25519Sha512, SignatureScheme},
+};
```

Comment:
In one of the PRs Egor introduced Crypto module. Would be great to reuse it here as well

Reply:
@kamilsa I think you talk about https://github.com/hyperledger/iroha/pull/465 - this is the same module (look at the file name)